go.mod

```diff
@@ -0,0 +1,23 @@
+module github.com/kubernetes-csi/external-health-monitor
+
+go 1.12
+
+require (
+	github.com/container-storage-interface/spec v1.2.0-rc1.0.20200415022618-e129a75169c1
```

Comment:
Does this have the latest changes we need for VolumeCondition?  Should we use master for now?